webull.py

```diff
@@ -433,8 +466,15 @@ def cancel_otoco_order(self, order_id=''):
     '''
     get price quote
     '''
-    def get_quote(self, stock='') :
-        response = requests.get(urls.quotes(self.get_ticker(stock)))
+    def get_quote(self, stock=None, tId=None) :
+        if not stock is None:
```

Comment:
If you do the other check first, can save an unnecessary function call in the case the caller provided both arguments.
if not tId is None:
    pass
elif not stock is None:
    tId = self.get_ticker(stock)
else:
    raise ValueError....

Reply:
Roger, thks for feedback